docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,652 @@
+- This `CONNECTION_REDIS_URL` environment variable is populated by a Kubernetes Secret.
+
+You can deploy this application for the first time by following these steps:
+
+- Create the Kubernetes namespace `demo`
+- Create the Kubernetes Secret `redis-secret` containing the Redis URL.
+- Install the Helm chart.
+
+{{< alert title="💡 Redis" color="info" >}}
+For now you're not going to actually deploy Redis, the URL in this step is fake. You will deploy Redis using a Recipe later in the tutorial that will replace the fake URL contained within `redis-secret`.
+{{< /alert >}}
+
+Complete these steps by running the following commands:
+
+```bash
+kubectl create namespace demo
+kubectl create secret generic --namespace demo --from-literal=url=redis://fake-server redis-secret 
+helm upgrade demo ./Chart -n demo --install
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl create namespace demo
+namespace/demo created
+
+> kubectl create secret generic --namespace demo --from-literal=url=redis://redis-master:6379 redis-secret 
+secret/redis-secret created
+
+> helm upgrade demo ./Chart -n demo --install
+Release "demo" does not exist. Installing it now.
+NAME: demo
+LAST DEPLOYED: Wed Sep 13 01:05:19 2023
+NAMESPACE: demo
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+{{< alert title="⚠️ Chart Directory" color="warning" >}}
+If you see an error message like **Error: path "./Chart" not found** then you are in the wrong directory. Make sure your terminal is in the `./demo` directory of the `samples` repository.
+{{< /alert >}}
+
+Run the following command to check if everything is running:
+
+```bash
+kubectl get all -n demo
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl get all -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/webapp-79d5dfb99-vhj9g   1/1     Running   0          2m48s
+
+NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/webapp   1/1     1            1           2m49s
+
+NAME                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/webapp-79d5dfb99   1         1         1       2m49s
+```
+
+> The generated names and ages of the objects will be different in your output. Make sure you see the status of `Running` for the `pod/webapp-...` entry. If the status is not `Running`, try repeating the `kubectl get all -n demo` after waiting.
+
+At this point you've deployed the application but you have not actually used Radius yet. You will start doing that in the next step, as well as set up and use Redis.
+
+The steps so far are similar to how many applications are managed today:
+
+- ⬜️ Dependencies like Redis are provisioned manually and separately from application deployment.
+- ⬜️ Connection information like passwords and addresses is manually stored in secret stores.
+- ⬜️ Applications access the connection information from those secret stores when they are deployed.
```

Comment:
What are the white boxes supposed to signify? We can probably delete these

Reply:
I added them to better align with the checkmark emojis at the bottom -- I suppose I meant for the white boxes to represent blank checkboxes. Given that, do you think we should remove these? Should we remove the green checkbox emojis below too?

Reply:
Hmm my preference is no checkboxes here but keep the green checks below, but not a strong opinion. Thanks!

Reply:
removing!